docs/tailwind.config.js

```diff
@@ -3,7 +3,11 @@ export default {
     "./src/**/*.{html,js,jsx,ts,tsx}",
   ],
   theme: {
-    extend: {},
+    extend: {
+      maxWidth: {
+        '8xl': '88rem',
```

Comment:
@tshuck-beyond Noticed we were using `max-w-8xl` within the readme directory, but that class does not come with Tailwind by default, so added it here.